docs/crio.conf.5.md

```diff
@@ -222,6 +222,10 @@ the container runtime configuration.
 **manage_ns_lifecycle**=true
   Determines whether we pin and remove namespaces and manage their lifecycle.
 
+**drop_infra_ctr**=false
+  Determines whether we drop the infra container when a pod does not have a private PID namespace, and it is not a kernel separated container.
+  Requies manage_ns_lifecycle to be true
```

Comment:
nit: `s/$/./` (missing `.` at EOL).

Reply:
nit: `s/manage_ns_lifecycle/**manage_ns_lifecycle**/`

Reply:
> a kernel separated container

I'm not sure I know what that means :thinking:

Reply:
fixed all!